Lib/heapq.py

```diff
@@ -314,7 +314,7 @@ def _siftup_max(heap, pos):
     _siftdown_max(heap, startpos, pos)
 
 def merge(*iterables, key=None, reverse=False):
-    '''Merge multiple sorted inputs into a single sorted output.
+    """Merge multiple sorted inputs into a single sorted output.
```

Comment:
Sorry, but I think that's no makes sense.

Reply:
The reasoning is based on observing that all the other functions in the module use double quotes for their docstrings.

[PEP 257](https://peps.python.org/pep-0257/) also mentions "For consistency, always use `"""triple double quotes"""` around docstrings."